web/src/HUD.tsx

```diff
@@ -430,7 +430,6 @@ class HUD extends Component<HudProps, HudState> {
       return (
         <Sidebar isClosed={isSidebarClosed} toggleSidebar={this.toggleSidebar}>
           <SidebarKeyboardShortcuts
-            history={this.props.history}
```

Comment:
why did this go away?

Reply:
i switch to using the useHistory() hook, which makes the component a bit more modular, https://github.com/ReactTraining/react-router/blob/15bb45b4d309bc3393659c4f4ded9bb55a03048b/packages/react-router/docs/api/hooks.md#usehistory